src/SFA.DAS.EAS.Web.IntegrationTests/EmployerAccountControllerTests/WhenGetLegalEntitiesWithNonExistentKey.cs

```diff
@@ -47,17 +48,18 @@ public async Task ThenTheStatusShouldBeFound()
             // Arrange
             const string accountName = "ACME Fireworks";
             const string legalEntityName = "RoadRunner Pest Control";
+            const string payeReference = "Acme PAYE";
 
-            var builder = _tester.DbBuilder;
-            builder
-                .BeginTransaction()
-                .EnsureUserExists(builder.BuildUserInput())
-                .EnsureAccountExists(builder.BuildEmployerAccountInput(accountName))
-                .WithLegalEntity(builder.BuildEntityWithAgreementInput(legalEntityName))
-                .CommitTransaction();
-            
+            string hashedAccountId;
+            using (var testDbContext = _tester.GetTransientInstance<EmployerAccountsDbBuilder>())
```

Comment:
Do you think the name testDbContext might be misinterpreted by an observer because of it's strong association to  EF's DbContext?  Something like dbBuilder might not have this.

Reply:
Agreed I have changed name